core/renderers/zelos/drawer.js

```diff
@@ -143,8 +150,31 @@ Blockly.zelos.Drawer.prototype.drawFlatBottom_ = function() {
  * @override
  */
 Blockly.zelos.Drawer.prototype.drawInlineInput_ = function(input) {
-  // Don't draw an inline input.
   this.positionInlineInputConnection_(input);
+
+  var hasConnection = !!input.connection.targetConnection;
+  var isInsertionMarker = input.connection.getSourceBlock().isInsertionMarker();
```

Comment:
Is this information already on the info object? And if not, should it be?  This is reaching out of rendering and into getting properties from the block directly.

Reply:
True, let me look into that.

Reply:
hasConnection can be replaced with ``input.connectedBlock``, but I don't see us using this elsewhere. I can add the meta data for whether the connectedBlock onto the input measurable.